rosdep/python.yaml

```diff
@@ -2562,7 +2562,7 @@ python-objectpath-pip:
       packages: [objectpath]
 python-omniorb:
   arch: [omniorbpy]
-  debian: [python-omniorb, python-omniorb-omg, omniidl-python]
+  debian: [omniorb, omniidl]
```

Comment:
I don't see any Python files in `omniorb`. Is `omniidl` enough here?

https://packages.debian.org/sid/amd64/omniorb/filelist
https://packages.debian.org/sid/amd64/omniidl/filelist

It looks like `python-omniorb` was a key in older versions of debian: https://tracker.debian.org/pkg/python-omniorb

I'm not really seeing the key in fedora either: https://src.fedoraproject.org/rpms/omniORB/blob/rawhide/f/omniORB.spec

Maybe it's time we drop this key in favor of https://github.com/ros/rosdistro/blob/10c5cc83d334bc32d1c614a0ef341b3b6ca0f343/rosdep/base.yaml#L6857 ?

Reply:
Looking at the link from @sloretz , it looks like this key was dropped from Debian in 2018.  So I think we can just remove this key completely.